src/dfx/src/lib/diagnosis.rs

```diff
@@ -17,10 +17,6 @@ pub type Diagnosis = (Option<String>, Option<String>);
 pub const NULL_DIAGNOSIS: Diagnosis = (None, None);
 
 #[derive(ThisError, Debug)]
-// This message will appear in the context trace of the stack. The diagnosis should not be displayed there yet.
-#[error("Diagnosis was added here.")]
-/// If you do not need the generic error diagnosis to run, you can add a DiagnosedError with .context(err: DiagnosedError).
-/// In that case, no extra diagnosis is attempted and the last-added explanation and suggestion are printed out.
```

Comment:
Why remove these comments? They describe the suggested usage of DiagnosedError.

Reply:
Fixed at https://github.com/dfinity/sdk/pull/4019/commits/0c9b12feb620103990f5033b8163447e3122748d